Assets/MRTK/Providers/WindowsMixedReality/Shared/InputHandlers/HPMotionControllerInputHandler.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using Unity.Profiling;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Input;
+
+//#if HP_CONTROLLER_ENABLED
```

Comment:
Testing this out without the new API package leads to a bunch of compilation errors in this class, I think due to these `#if` statements being commented out